### PR TITLE
Mattb/beta 1.1 updates

### PR DESCRIPTION
**Pull request by mattbradburyr3**
Updating the getting started docs for beta 1.1

### CODE REVIEW

File: content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md [hunk 1]
```diff
@@ -43,10 +44,10 @@ The images in this section show the `CSDE-cordapp-template-kotlin` repository in
    {{% /tab %}}
    {{< /tabs >}}
 
-2. Change to the new directory and checkout the Developer Preview 2 branch:
+2. Change to the new directory and checkout the beta 1 branch:
```
Comment (yarivus) [inline content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md:47]:
```suggestion
2. Change to the new directory and checkout the Beta 1 branch:
```
Or ...
```suggestion
2. Change to the new directory and checkout the 'release/corda-5-beta-1' branch:
```

File: content/en/platform/corda/5.0-beta/developing/getting-started/fast-feedback-with-the-simulator/fast-feedback-with-the-simulator.md [hunk 2]
```diff
@@ -8,6 +8,11 @@ menu:
     weight: 4000
 section_menu: corda-5-beta
 ---
+
+{{< note >}}
+As of Beta 1.1 Simulator does not support the UTXOLedgerService, for now this limits the usefulness of the simulator for utxo based application.
```
Comment (yarivus) [inline content/en/platform/corda/5.0-beta/developing/getting-started/fast-feedback-with-the-simulator/fast-feedback-with-the-simulator.md:13]:
```suggestion
As of Beta 1.1 Simulator does not support the UTXOLedgerService, for now this limits the usefulness of the simulator for UTXO based application.
```

**Review by yarivus — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md [hunk 3]
```diff
@@ -72,45 +73,74 @@ This section provides an overview of the content of CSDE. Other sections show yo
...
 The **combined worker** is a Corda cluster that runs all of the workers in one JVM process.
 
-#### `startCorda`
 
-The `startCorda` task does the following:
+The helpers are split into three folders:
+
+#### csde-corda
+
+These task help with the lifecycle of your local Corda Cluster.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md:98]:
```suggestion
These tasks help with the lifecycle of your local Corda cluster.
```

File: content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md [hunk 4]
```diff
@@ -72,45 +73,74 @@ This section provides an overview of the content of CSDE. Other sections show yo
...
 1. Downloads and locally stores a copy of the combined worker JAR, if required
 2. Starts an instance of a Postgres Docker container; you will need Docker Engine or Docker Desktop running
 3. Starts the combined worker
 
-#### `stopCorda`
+Note, corda takes around a minute to start up, however at present the gradle tasks shows as complete before the start up has finished. It is best to poll the cluster with one of the csde-query helpers until it responds, this will tell you the cluster is live and ready to interact.
+
+`stopCorda` which:
+
+#### `startCorda`
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md:109]:
```suggestion
```

File: content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md [hunk 5]
```diff
@@ -72,45 +73,74 @@ This section provides an overview of the content of CSDE. Other sections show yo
...
+1. Stops the Postgres database
+2. Stops the combined worker
+
+#### csde-cordapp
+
+These tasks help with the compilation and deployment of your Cordapp.
+
+`1-createGroupPolicy` creates the Group policy which is required to set up the Application Network.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md:117]:
```suggestion
`1-createGroupPolicy` creates the Group policy which is required to set up the application network.
```

File: content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md [hunk 6]
```diff
@@ -72,45 +73,74 @@ This section provides an overview of the content of CSDE. Other sections show yo
...
-* Stops the Postgres database
-* Stops the combined worker
+`4-deployCPIs` deploys the CPI to your local corda cluster.
 
-#### `deployCorDapp`
+`5-createAndRegisterVNodes` sets up the Virtual Nodes specified in `config/dev-net.json` on you local corda with the uploaded CPI
+
+Each of these tasks has a dependency on the previous, so if you run 3, it will also run 1 and 2.
+
+`quickDeployCordapp` is just an alias for `5-createAndRegisterVNodes`, you should click this to do your first deployment of your Cordapp to a newly started cluster.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md:129]:
```suggestion
`quickDeployCordapp` is just an alias for `5-createAndRegisterVNodes`. You should click this to do your first deployment of your CorDapp to a newly started cluster.
```

File: content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md [hunk 7]
```diff
@@ -72,45 +73,74 @@ This section provides an overview of the content of CSDE. Other sections show yo
...
+
+{{< note >}}
+You only need to run `quickDeployCordapp` the first time you upload your CPI to the corda cluster. On subsequent builds you can just click `4-deployCPIs`. This will reduce your turnaround time for deploying your code from 1-2 mins to around 20-30 seconds.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md:132]:
```suggestion
You only need to run `quickDeployCordapp` the first time you upload your CPI to the corda cluster. On subsequent builds, you can just run `4-deployCPIs`. This reduces your turnaround time for deploying your code from 1-2 mins to around 20-30 seconds.
```

**Review by yarivus — DISMISSED** [review]
LGTM

File: content/en/platform/corda/5.0-beta/developing/getting-started/installing-corda-cli.md [hunk 8]
```diff
@@ -24,21 +24,25 @@ Java     | Azul JDK 11
 
 ## Downloading Corda CLI
 
-To obtain the Corda CLI installer:
+To obtain the Corda CLI installer from the release pack on Customer Hub:
 1. Download `platform-jars-Fox.tar.gz` from the [R3 Customer Hub](https://r3.force.com/).
-2. Extract `corda-cli-installer-5.0.0.0-Fox1.0.zip` from `net\corda\cli\deployment\corda-cli-installer\5.0.0.0-Fox1.0`.
+2. Extract `corda-cli-installer-5.0.0.0-Fox1.1.zip` from `net\corda\cli\deployment\corda-cli-installer\5.0.0.0-Fox1.1`.
+
+To Obtain the Corda Cli installer without the release pack:
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/installing-corda-cli.md:31]:
```suggestion
To obtain the Corda CLI installer without the release pack:
```

Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/installing-corda-cli.md:31]:
```suggestion
```

File: content/en/platform/corda/5.0-beta/developing/getting-started/installing-corda-cli.md [hunk 9]
```diff
@@ -24,21 +24,25 @@ Java     | Azul JDK 11
 
 ## Downloading Corda CLI
 
-To obtain the Corda CLI installer:
+To obtain the Corda CLI installer from the release pack on Customer Hub:
 1. Download `platform-jars-Fox.tar.gz` from the [R3 Customer Hub](https://r3.force.com/).
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/installing-corda-cli.md:28]:
```suggestion
1. Download `platform-jars-Fox.tar.gz` from the [R3 Customer Hub](https://r3.force.com/) or from the [R3 S3 repository](https://download.corda.net/packages/corda-cli-downloader/5.0.0.0-Fox1.1/corda-cli-downloader-5.0.0.0-Fox1.1.zip).
```

File: content/en/platform/corda/5.0-beta/developing/getting-started/installing-corda-cli.md [hunk 10]
```diff
@@ -24,21 +24,25 @@ Java     | Azul JDK 11
...
-2. Extract `corda-cli-installer-5.0.0.0-Fox1.0.zip` from `net\corda\cli\deployment\corda-cli-installer\5.0.0.0-Fox1.0`.
+2. Extract `corda-cli-installer-5.0.0.0-Fox1.1.zip` from `net\corda\cli\deployment\corda-cli-installer\5.0.0.0-Fox1.1`.
+
+To Obtain the Corda Cli installer without the release pack:
+1. Download `platform-jars-Fox.tar.gz` from the R3 S3 repository [here](https://download.corda.net/packages/corda-cli-downloader/5.0.0.0-Fox1.1/corda-cli-downloader-5.0.0.0-Fox1.1.zip).
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/installing-corda-cli.md:32]:
```suggestion
```

File: content/en/platform/corda/5.0-beta/developing/getting-started/installing-corda-cli.md [hunk 11]
```diff
@@ -24,21 +24,25 @@ Java     | Azul JDK 11
...
+2. Extract the files.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/installing-corda-cli.md:33]:
```suggestion
```

File: content/en/platform/corda/5.0-beta/developing/getting-started/running-your-first-cordapp/run-first-cordapp.md [hunk 12]
```diff
@@ -21,7 +21,7 @@ The `startCorda` task should complete relatively quickly with this message:
 {{< figure src="starting-corda-complete.png" figcaption="CSDE startCorda task completed" alt="CSDE task to start the combined worker in IntelliJ completed" >}}
 
 However, you must wait approximately one minute for Corda to finish its start-up routine.
-Currently, we do not have a liveness detector for Corda in the CSDE so we check liveness by manually hitting an endpoint.
+Currently, we do not have a liveness detector for Corda in the CSDE so we check liveness by manually hitting an endpoint. You can use the listVNodes helper for this.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/running-your-first-cordapp/run-first-cordapp.md:24]:
```suggestion
Currently, we do not have a liveness detector for Corda in the CSDE so we check liveness by manually hitting an endpoint. You can use the [`listVNodes` helper](cordapp-standard-development-environment/csde.html#csde-queries) to do this.
```

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md [hunk 13]
```diff
@@ -72,45 +73,74 @@ This section provides an overview of the content of CSDE. Other sections show yo
...
+
+These task help with the lifecycle of your local Corda Cluster.
 
+`startCorda` which:
 1. Downloads and locally stores a copy of the combined worker JAR, if required
 2. Starts an instance of a Postgres Docker container; you will need Docker Engine or Docker Desktop running
 3. Starts the combined worker
 
-#### `stopCorda`
+Note, corda takes around a minute to start up, however at present the gradle tasks shows as complete before the start up has finished. It is best to poll the cluster with one of the csde-query helpers until it responds, this will tell you the cluster is live and ready to interact.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md:105]:
```suggestion
Note, Corda takes about one minute to start up. However, at present, the Gradle task shows as complete before the start up has finished. It is best to poll the cluster with one of the `csde-query` helpers until it responds to confirm if the cluster is live and ready to interact.
```

File: content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md [hunk 14]
```diff
@@ -72,45 +73,74 @@ This section provides an overview of the content of CSDE. Other sections show yo
...
+#### csde-cordapp
+
+These tasks help with the compilation and deployment of your Cordapp.
+
+`1-createGroupPolicy` creates the Group policy which is required to set up the Application Network.
+
+`2-createKeyStore` creates the signing keys for publishing the CPIs.
 
-The `stopCorda` task does the following:
+`3-buildCPIs` builds your Cordapp and wraps it in a signed CPI.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md:121]:
```suggestion
`3-buildCPIs` builds your CorDapp and wraps it in a signed CPI.
```

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**